iothub/device/src/Transport/Mqtt/MqttIotHubAdapter.cs

```diff
@@ -1149,8 +1151,8 @@ public static async Task WriteMessageAsync(IChannelHandlerContext context, objec
                 }
             }
         }
-               
-        static string PopulateMessagePropertiesFromMessage(string topicName, Message message)
+
+        internal string PopulateMessagePropertiesFromMessage(string topicName, Message message)
```

Comment:
Was this supposed to be `internal static` ? (Just properly specifying that the method is internal - i.e. no real change?)